File: app/src/main/java/com/example/player/a1610aplayerdemo/MyApplication.java
package com.example.player.a1610aplayerdemo;

import android.app.Application;
import android.content.Intent;
import com.example.player.a1610aplayerdemo.ui.my.login.LoginActivity;
import com.example.player.a1610aplayerdemo.ui.my.login.LoginPresenter;
import com.example.player.a1610aplayerdemo.utils.TokenSp;
import com.example.player.a1610aplayerdemo.utils.UserSp;
import com.umeng.commonsdk.UMConfigure;

public class MyApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        TokenSp.getInstance().init(this);
        UMConfigure.init(this,UMConfigure.DEVICE_TYPE_PHONE,null);
        UserSp.getInstance().init(this);
    }
}

File: app/src/main/java/com/example/player/a1610aplayerdemo/util/MyApplication.kt
package com.example.player.a1610aplayerdemo.util

import android.app.Application
import android.content.Context

class MyApplication: Application() {

    companion object {
        var context:Application? = null
        fun getContext():Context{
            return context!!
        }
    }

    override fun onCreate() {
        super.onCreate()
        context = this
    }

}